### ISSUE TITLE
Elevated urine pyridinoline concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6027]
**Preferred term label:** Elevated urine pyridinoline concentration

**Synonyms** Elevated urine PYD concentration

**Definition (free text, please give PubMed ID)** Concentration of pyridinoline in the urine is above the normal range., PMID: 9127470; PMID: 11350019

**Parent term (use hpo.jax.org/app)** Abnormality of urine homeostasis HP:0003110

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://www.ebi.ac.uk/ols/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_88539

**Comment by pnrobinson** [comment]
added
